Makefile

```diff
@@ -9,6 +9,9 @@ all: $(addprefix bin/, $(BINARIES))
 clean:
 	$(RM) -r bin/ _artifact/
 
+bin/rootlesskit-debug: $(GO_FILES)
```

Comment:
This should be a separate PR, and I'd rather prefer to use a variable like `GO_BUILD_FLAGS`

Reply:
agree